webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -603,91 +603,81 @@ private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, Scram
         if(showScrambleCount) {
             offsetTop -= fontSize + 2;
         }
-
-        // The 100 number in the fit text function is just some big number. Hopefully, fitting the width will be enough to fit the height.
-        Rectangle rect = new Rectangle(competitorInfoLeft+(right-competitorInfoLeft)/2, top-offsetTop, right-competitorInfoLeft, 100);
-
-        fitAndShowText(cb, globalTitle, bf, rect, fontSize, PdfContentByte.ALIGN_CENTER);
+        
+        Rectangle rect = new Rectangle(competitorInfoLeft, top-offsetTop+fontSize, right, top-offsetTop);
+        fitAndShowText(cb, globalTitle, bf, rect, fontSize, Element.ALIGN_CENTER, 1);
```

Comment:
> fitAndShowText(cb, globalTitle, bf, rect, fontSize, Element.ALIGN_CENTER, **1**);

@jfly I was wondering. This 1 here is the leading multiplier. It's not necessary here since it's a 1 line text (just like a text with simple line spacing). Would you rather a new variable or a new function that doesn't take this as param?